operator/hack/update-generated-crd.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+function install_crd {
+  if [[ -z "${SKIP_COPY+1}" ]]; then
+    local SRC="$1"
+    local DST="$2"
+    if ! cmp -s "$DST" "$SRC"; then
+      cp -f "$SRC" "$DST"
+    fi
+  fi
+}
+
+# Can't rely on associative arrays for old Bash versions (e.g. OSX)
+install_crd \
+  "vendor/github.com/openshift/api/network/v1alpha1/0000_70_dnsnameresolver_00-techpreview.crd.yaml" \
```

Comment:
The api repository changed recently. It moved the generated CRDs into a separate package: `zz_generated.crd-manifests`. Do we have a CI to catch this?

Reply:
We are currently using an older version for openshift/api which will match with that of cluster-dns-operator. As we are importing the DNSNameResolver controller into CDO, any mismatch in the version in the go.mod file will cause the version to change in the go.mod file of the CDO.